src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/wrapper/PKCS11.java

```diff
@@ -112,6 +112,9 @@ public static void loadNative() {
      */
     private final String pkcs11ModulePath;
     private final CK_VERSION version;
+
+    // Note: Please don't update this field other than the constructor.
+    // Otherwise, the native data is not able to be collected.
```

Comment:
If it is only set in the constructor, it can be 'final".

Reply:
I would prefer to use 'final'.  But this field is set by JNI code.  I'm not very sure of the impact if declaring it as final field.